package.json

```diff
@@ -127,9 +126,10 @@
     "react-native-popup-menu": "0.13.3",
     "react-native-push-notification": "^3.0.2",
     "react-native-qrcode": "^0.2.3",
-    "react-native-randombytes": "^3.5.0",
-    "react-native-router-flux": "4.0.0-beta.31",
-    "react-native-share": "^1.1.1",
+    "react-native-randombytes": "git://github.com/Airbitz/react-native-randombytes.git",
```

Comment:
why did anything in package.json change other than `react-native-size-matters`?

Reply:
I am not really sure why

Reply:
Please rebase this PR without the incorrect changes to package.json